src/shared/components/data-table/components/main/components/table-header/TableHeader.scss

```diff
@@ -0,0 +1,69 @@
+@import 'src/styles/common';
+
+.header {
+  height: 30px;
+  margin-top: 0px;
```

Comment:
As opposed to what? Is this rule doing anything?

Reply:
Leftovers from some experiments I did previously. Removed it